pkg/codegen/docs/gen.go

```diff
@@ -1145,15 +1151,21 @@ func (mod *modContext) genResource(r *schema.Resource) resourceDocArgs {
 	if !r.IsProvider {
 		filteredOutputProps = filterOutputProperties(r.InputProperties, r.Properties)
 	}
-	hasFilteredOutputProps := len(filteredOutputProps) > 0
+
+	// All resources have an implicit `id` output property, that we must inject into the docs.
+	filteredOutputProps = append(filteredOutputProps, &schema.Property{
+		Name:       "id",
```

Comment:
The css class name for this property's `<dt>` tag is being generated as `property-`. I guess it's fine, yeah? Or should we decorate the property somehow uniquely?

Reply:
I noticed this as well, and it's already true for all the output properties. I was thinking about cleaning that up in a separate change, but since it has no visible impact, seemed like a lower priority change.